src/server/pkg/obj/obj.go

```diff
@@ -26,6 +26,11 @@ type Client interface {
 	Delete(name string) error
 	// Walk calls `fn` with the names of objects which can be found under `prefix`.
 	Walk(prefix string, fn func(name string) error) error
+	// Exsits checks if a given object already exists
+	Exists(name string) bool
+	// IsBenevolent determines if an error is benevolent, i.e. that it
+	// can be ignored.
+	IsBenevolent(err error) bool
```

Comment:
I'd vote we name this `IsIgnorable` then the information in this comment would be implicitly understood.